_applications.md

```diff
@@ -32,11 +32,12 @@ transactions:transfer:self        | Can create a [transaction](#transaction-obje
 transactions:withdraw             | Can create a withdrawal [transaction](#transaction-object).
 user:read                         | Can view the [user](#user-object) and their information.
 
+### Deprecated scopes
 The following _scopes_ are deprecated and will be removed in a future version of the API:
 
-Scope                 | Description
---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-transactions:write                | Can create a [transaction](#transaction-object) from any card to any destination (another card or an external address), cancel and resend transactions. This scope is now deprecated in favor of the more fine-grained write scopes above (deposit, transfer and withdraw).
+Scope              | Description
+------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+transactions:write | Can create a [transaction](#transaction-object) from any card to any destination (another card or an external address), cancel and resend transactions. This scope is now deprecated in favor of the more fine-grained write scopes above (deposit, transfer and withdraw).
```

Comment:
I'd replace:

    from any card to any destination (another card or an external address), …

with:

    from any origin to any destination, …